apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,8 +1,8 @@
 ack_generate_info:
-  build_date: "2022-06-09T17:20:43Z"
-  build_hash: 6acf40fe3e3cfd97b799ef7cbf1e89e01c3db8f7
+  build_date: "2022-08-02T03:26:17Z"
+  build_hash: fe61d04673fd4d9848d5f726b01e0689a16d3733
   go_version: go1.18.3
-  version: v0.18.4-15-g6acf40f
+  version: v0.18.4-37-gfe61d04-dirty
```

Comment:
PR looks good, just checking why the code-generator version here is not `v0.19.4`